package.json

```diff
@@ -68,8 +68,7 @@
     "@storybook/addon-actions": "5.1.9",
     "@storybook/html": "5.1.9",
     "@storybook/theming": "5.1.9",
-    "@types/algoliasearch": "3.30.14",
-    "@types/algoliasearch-helper": "2.26.1",
```

Comment:
was no longer required since we used the canary version of the helper, but it seems like it was merged back in wrongly